src/it/java/teammates/it/storage/sqlapi/FeedbackQuestionsDbIT.java

```diff
@@ -84,4 +87,45 @@ public void testHasFeedbackQuestionsForGiverType() {
 
         assertTrue(actual);
     }
+
+    private FeedbackQuestion prepareSqlInjectionTests() {
+        FeedbackQuestion fq = typicalDataBundle.feedbackQuestions.get("qn1InSession1InCourse1");
+        fqDb.createFeedbackQuestion(fq);
+
+        // Ensure feedback_questions db has at least 1 entry / row.
+        assertNotNull(fqDb.getFeedbackQuestion(fq.getId()));
+
+        return fq;
+    }
+
+    @Test
+    public void testSqlInjectionInCreateFeedbackQuestion() {
+        prepareSqlInjectionTests();
+
+        FeedbackSession fs = typicalDataBundle.feedbackSessions.get("session1InCourse1");
+        String maliciousDescription = "', '', '', 1, '', '', '', ''); DELETE FROM feedback_questions;--";
```

Comment:
I just updated the fields but it can be confirmed by checking the logged sql statements from hibernate. 
<img width="1144" alt="Screenshot 2024-02-27 at 7 26 49 PM" src="https://github.com/TEAMMATES/teammates/assets/77243938/59f41639-7a1c-4373-9f0c-cdd621febc9c">

Reply:
Actually, I wonder if it's possible that doing this is not necessary. If SQL injection were possible and it was not in that order or the number of columns remaining was wrong, then an error should be raised. If there was no error raised, either SQL injection is not possible (yay), or it was in the correct order. But even if it was in the correct order, then the following code asserts that the SQL injected query did not work. Unless you spot a flaw in this logic?

I'm sorry that you went to make the changes already.